frappe/desk/desktop.py

```diff
@@ -194,6 +194,9 @@ def _prepare_item(self, item):
 
 				item["count"] = count
 
+		if item.get("link_type") == "DocType":
+			item["description"] = frappe.get_meta(item.link_to).description
```

Comment:
meta is cached, so this is fast enough IMO. Is the entire final workspace cached? If not we should do it.

Reply:
Just maintains these sidebar pages and sends these on subsequent requests while switching workspaces

https://github.com/frappe/frappe/blob/1b8e9c3502b702178f777cf2e5c253ccec573e2e/frappe/public/js/frappe/views/workspace/workspace.js#L68

Doesn't cache the entire workspace on the server side it seems. This can be addressed in a separate PR IG. Can this go in first?